pkg/pb/inspector.proto

```diff
@@ -47,6 +47,29 @@ service PieceStoreInspector {
   rpc Dashboard(DashboardRequest) returns (DashboardResponse) {}
 }
 
+service IrreparableInspector {
+  // ListSegments returns damaged segments
+  rpc ListSegments(ListSegmentsRequest) returns (ListSegmentsResponse);
+}
+
+// ListSegments
+message ListSegmentsRequest {
+  int32 limit = 1;
+  int32 offset = 2;
+}
+
+message IrreparableSegment {
```

Comment:
If IrreparableSegment is nearly an identical clone of RemoteSegmentInfo, I suggest refactoring to remove RemoteSegmentInfo.  While you're at it, life would probably be better if you renamed away from 'Encrypted' (I don't think they are), and also if you stored EncryptedSegmentDetail as a pb.Pointer directly so it automatically marshals.